MAINTAINERS.md

```diff
@@ -0,0 +1,144 @@
+# SPIRE Maintainership Guidelines and Processes
+This document captures the values, guidelines, and processes that the SPIRE project and its maintainers adhere to. All SPIRE maintainers, in their independent and individual capacity, agree to uphold and abide by the text contained herein.
+
+This process can be changed, either permanently or as a one-time exception, through an 80% supermajority maintainer vote.
+
+For a list of active SPIRE maintainers, please see the [CODEOWNERS](CODEOWNERS) file.
+
+## General Governance
+The SPIRE project abides by the same [governance procedures][1] as the SPIFFE project, and ultimately reports to the SPIFFE TSC the same way that the SPIFFE project and associated maintainers do.
+
+TSC members do not track day-to-day activity in the SPIFFE/SPIRE projects, and this should be considered when deciding to raise issues to them. While the SPIFFE TSC has the ultimate say, in practice they are only engaged upon serious maintainer disagreement. To say that this would be unprecedented is an understatement.
+
+### Maintainer Responsibility
+SPIRE maintainers adhere to the [requirements and responsibilities][2] set forth in the SPIFFE governance document. They further pledge the following:
+* To act in the best interest of the project at all times
+* To ensure that project development and direction is a function of real user need
+* To never take an important action in maintainer capacity with hesitation
```

Comment:
this phrasing feels a bit awkward to me, maybe it'd be more clear to say `To never take any action which would result in an uncertain or potentially damaging outcome`. just something about `with hesitation` feels off to me but i can't put it into words exactly

Reply:
I'll reword a bit. I want to keep the spirit of the "with hesitation", which is to say, a maintainer shouldn't perform an action while being hesitant that it is the right thing to do.